go_attack_utils/src/sgf_parser/game_info.py

```diff
@@ -195,12 +201,17 @@ def parse_game_str_to_dict(
     if win_color is None:
         adv_minus_victim_score = 0
     else:
-        win_score = (
-            float(result.split("+")[-1])
+        win_score_str = (
+            result.split("+")[-1]
             if "+" in result
             # Sgfs for manual games can have a space instead of a +
-            else float(result.split(" ")[-1])
+            else result.split(" ")[-1]
         )
+        if win_score_str == "R":
+            # Resignation
+            win_score = 0
```

Comment:
0 is a pretty weird `win_score` though I'm admittedly not sure what's better. Perhaps `None` if this doesn't result in a crash downstream.